jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/test/java/org/jbpm/workbench/ht/client/editors/taskslist/AbstractTaskListViewTest.java

```diff
@@ -73,7 +73,7 @@ public void initColumnsWithTaskVarColumnsTest() {
             @Override
             public Void answer(InvocationOnMock invocationOnMock) throws Throwable {
                 final List<ColumnMeta> columns = (List<ColumnMeta>) invocationOnMock.getArguments()[0];
-                assertTrue(columns.size() == 18);
+                assertTrue(columns.size() == 19);
```

Comment:
Can you please replace _assertTrue_ method with _assertEquals_, in the way it's used in _DataSetDefsBootstrapTest_ where you did the argument modification from 19 to 20?
If the assertion fails, the assertion error returned is much clearer in that case.

Reply:
Done